otherlibs/dune-rpc/dune_rpc.mli

```diff
@@ -86,6 +86,112 @@ module V1 : sig
     val to_string_absolute : t -> string
   end
 
+  module Ansi_color : sig
+    module RGB8 : sig
+      (** 8-bit RGB color *)
+      type t
+
+      (** [of_int i] creates an 8-bit color from the least significant 8 bits of [i]. *)
+      val of_int : int -> t
+
+      (** [to_int t] returns the 8-bit color as an integer in the range [0, 255]. *)
+      val to_int : t -> int
```

Comment:
I don't think these functions are very useful. What about accessors like `red`, `green`, `blue`?

Reply:
It's not possible to do split into red, green and blue like we do for 24-bit color since not every terminal encodes 8-bit colors in the same manner. The only thing that doesn't change is that there are 256 colors.

Reply:
The only time this can happen is if `Dune` runs a program with 8-bit color output and we wish to preserve that color. At that point we are at the mercy of the client on how we quantize those values.